### PR TITLE
Make a minor update to the `FileInfo` table formatting on Unix to make it more concise

### PR DESCRIPTION
**Pull request by daxian-dbw**
# PR Summary

Make a minor update to the `FileInfo` table formatting on Unix to make it a little more concise.
The existing formatting has too much blank space in the table, which makes it a little hard to quick scan the file name column, which is at the right most.

A few changes to the view:
1. Right-align the `User` column, so that `User` and `Group` are closely together.
2. Change the width of `User` and `Group` to be 10 instead of 16. The old limit to username (like in Ubuntu 11.04) was 8 characters, and now the limit is 32 characters. I did a search on _common Linux usernames_, and most of the names I found falls under 10 characters. So, change the width to be 10 instead of 16 to reduce blank spaces of the table view.
3. `LastWriteTime` uses a fixed format for this view, so we can use the exact width, which is 16.
4. For the `Size` column or (length), using width 12 will allow us to show a file size of `931 gb` without truncating, which should be sufficient in most cases.

## PR Context

Before the change:

image

After the change:

image

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/7.3/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.

### CODE REVIEW

File: src/System.Management.Automation/FormatAndOutput/DefaultFormatters/FileSystem_format_ps1xml.cs [hunk 1]
```diff
@@ -46,10 +46,10 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_FileSystemTypes(CustomC
...
                     .GroupByProperty("PSParentPath", customControl: sharedControls[0])
                     .AddHeader(Alignment.Left, label: "UnixMode", width: 10)
-                    .AddHeader(Alignment.Left, label: "User", width: 16)
-                    .AddHeader(Alignment.Left, label: "Group", width: 16)
-                    .AddHeader(Alignment.Right, label: "LastWriteTime", width: 18)
-                    .AddHeader(Alignment.Right, label: "Size", width: 14)
+                    .AddHeader(Alignment.Right, label: "User", width: 10)
+                    .AddHeader(Alignment.Left, label: "Group", width: 10)
+                    .AddHeader(Alignment.Right, label: "LastWriteTime", width: 16)
+                    .AddHeader(Alignment.Right, label: "Size", width: 12)
```
Comment (iSazonov) [inline src/System.Management.Automation/FormatAndOutput/DefaultFormatters/FileSystem_format_ps1xml.cs:52]: If we think about VMWare large files with size over 1Tb are a common occurrence. And they have PowerShell tools :smile:
```suggestion
                    .AddHeader(Alignment.Right, label: "Size", width: 14)
```

**Review by iSazonov — APPROVED** [review]
LGTM with one minor comment.

File: src/System.Management.Automation/FormatAndOutput/DefaultFormatters/FileSystem_format_ps1xml.cs (hunk 1 above)
Comment (iSazonov) [reply src/System.Management.Automation/FormatAndOutput/DefaultFormatters/FileSystem_format_ps1xml.cs:52]: And maybe use our new human formatting method we merged recently?

Comment (daxian-dbw) [reply src/System.Management.Automation/FormatAndOutput/DefaultFormatters/FileSystem_format_ps1xml.cs:52]: I understand large VM disk could be over TB, like VHDX can be up to 64TB. But I don't think that's a common thing in most daily uses, also the list view could be helpful if the user really wants to see the length displayed on screen.

> And maybe use our new human formatting method we merged recently?

Size here should be accurate, so we should not use that.

Comment (iSazonov) [reply src/System.Management.Automation/FormatAndOutput/DefaultFormatters/FileSystem_format_ps1xml.cs:52]: We can merge.

**Pull request merged**